src/app/migrations/forms/forms.3.core.txt

```diff
@@ -0,0 +1,34 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  template: `
+    <cds-form-group layout="compact" control-width="shrink">
```

Comment:
https://github.com/vmware/clarity/blob/eac9ab63065c2a2b0a9d3661e75d0d7b21b0f3dc/packages/core/src/forms/form-group/form-group.element.ts#L92 

`cds-form-group` doesn't act as a `form` by itself. It should be wrapped in a `<form>`